apps/accounts/utils/general.py

```diff
@@ -304,7 +309,7 @@ def get_user_email(user, email, is_confirmed=True):
     return user_email
 
 
-def associate_user_email(
+def associate_user_email(  # noqa: C901
```

Comment:
This is nuts

https://www.flake8rules.com/rules/C901.html

In other teams we have ignored rule C901

Reply:
Yes, I also find it very unintuitive